content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,241 @@
+
+This feature requires admins to set up executors (much like CI agents), which Sourcegraph will use to offload expensive tasks. Executors can also be used to run [code navigation auto-indexing](https://github.com/sourcegraph/about/blob/bf341aafde75450c295aaec064470017020cdcd8/content/blogposts/2022/sourcegraph-4.0.md#TODO-link-to-section-or-docs). All Sourcegraph Cloud instances have one or more executors available by default. 
+
+<Video 
+  source={{
+    webm: 'blog/release-post/4.0/Server%20Side%20Batch%20Changes%20',
+    mp4: 'blog/release-post/4.0/Server%20Side%20Batch%20Changes%20'
+  }}
+  loop={true}
+  title="Server-side Batch Changes"
+  caption="Large-scale changes can be run across thousands of repositories with server-side Batch Changes"
+/>
+
+<a href="https://docs.sourcegraph.com/batch_changes/explanations/server_side" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### ☁️ Dedicated Sourcegraph Cloud instances for enterprise
+
+<br />
+<a href="https://docs.sourcegraph.com/cloud"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Scale confidently with a dedicated and secure instance of Sourcegraph Cloud
+After almost a decade of developing self-hosted Sourcegraph, we’ve gained the trust of some of the most sophisticated companies and development teams in the world, allowing us to work with their most sensitive IP: their code. Today—after years of building trust and engineering rigor—we are excited to move confidently to the Cloud with a highly secure and scalable solution.
+
+Dedicated, single-tenant Sourcegraph Cloud instances are now generally available, and the best way for new teams to use Sourcegraph. You can read more in our announcement blog here[/blog/enterprise-cloud].
+
+<a href="https://docs.sourcegraph.com/cloud" className="tw-not-italic tw-flex tw-items-center tw-mb-sm">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+  
+<br />
+<a href="https://docs.sourcegraph.com/cloud"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### SOC 2 Type II certification for Sourcegraph Cloud
+Sourcegraph continues our commitment to the security of your most valuable IP—your code—through our security operations, compliance measures, and best practices. With our recent SOC 2 Type II attestation for Soucegraph Cloud, we met the high standards of the third-party audit, and you can be confident that Sourcegraph controls for the security and availability of your sensitive data. 
+
+<a href="https://security.sourcegraph.com/" className="tw-not-italic tw-flex tw-items-center">Docs<OpenInNewIcon className="tw-ml-xxs" size={18} /></a>
+
+### 📈 Advanced admin capabilities
+<br />
+<a href="https://docs.sourcegraph.com/admin/updates#multi-version-upgrades"><Badge text="Admin" color="violet" size="small" /></a>
+
+#### Save time upgrading to Sourcegraph 4.0 with multi-version upgrades
```

Comment:
@efritz you please review this by EOD today to ensure the content is accurate? Please keep in mind that we're focusing on the benefits and value to our customers/prospects within this release post.

Reply:
LGTM!